### ISSUE TITLE
Design Intent - Accessibility Feedback - Disability Benefits Crew - Core Form Team, 21-526ez, Side Navigation

### ISSUE THREAD

**Issue by jasonday** [department-of-veterans-affairs/va.gov-team#121035]
## Next Steps for the VFS team

_How did this touchpoint go? [Give feedback on the Collaboration Cycle](https://ows.io/qs/gb1dfnkl) at any time._

- [ ] **Assign** this ticket to the team member(s) responsible for addressing feedback provided by Platform.
- [ ] **Comment** on this ticket:
  - If the Platform reviewer has any **Thoughts/Questions** that require responses.
  - When **Must** feedback has been incorporated. As appropriate, link to any other GitHub issues or PRs related to this feedback.
  - When **Should/Consider** feedback has been incorporated, or if any feedback will not be addressed. As appropriate, link to any other GitHub issues or PRs related to this feedback.
- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members`.
- [ ] **Close the ticket any time** after you've finished reviewing it. You don't need to confirm with Governance before closing.

## Thoughts/questions

-

## Feedback
Practice areas will document their feedback on the VFS-provided artifacts following the [Must, Should, and Consider Framework](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/must-should-and-consider-framework-for-feedback). Platform Governance reviewers may also provide additional notes that don’t comment on the artifacts themselves but are important for implementation (eg. engineering/coding notes).

- [ ] **Should:** Each page as part of the flow, should have a unique page title that describes the purpose of the page. Current guidance for page titles does not reflect this guidance, however updates are currently in process. Page titles help users wayfind and orient themselves to the experience. When page titles are not unique, users, especially assistive technology users, may find it difficult to orient.
- [ ] **Consider:** As part of the presentation of steps of the flow, where some page links cannot be clicked due to a prior page being unfinished, consider providing text or some sort of affordance that indicates, for example, that step 4 is unfinished, so the user understands that they need to finish step 4 before being able to navigate to step 5. This indication may help older users and users with cognitive disabilities understand how to proceed in the flow and may prevent frustrated clicks on disabled links. 

## Governance team actions
- [x] Format feedback as individual tasks (check boxes)
- [x] Assign this ticket to the VFS team member that opened the Slack request
- [x] Add the VFS team product label
- [x] Add the VFS team the feature label (if applicable)
- [x] Add the touchpoint labels
- [x] Add the practice area labels
- [x] Add the Collaboration Cycle initiative milestone
- [ ] Add the 'collaboration-cycle-mobile-app' label if this feedback is for the mobile app

**Comment by BelovAndrew** [comment]
Follow up ticket created.
